### DISCUSSION TITLE
Allow customizing form separator fonts, text size, and color

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5629] [Feature Requests]
Supported attributes at present - https://github.com/specify/specify7/wiki/Form-System#separator

CSIRO requested being able to make text smaller and not bold

**Comment by grantfitzsimmons** [comment]
From https://github.com/specify/specify7/issues/2290

> It would be nice to be able to change the separator element color
> 
> **Requested By:** NMSU Herbaria
